src/quo2/components/buttons/dynamic_button.cljs

```diff
@@ -0,0 +1,103 @@
+(ns quo2.components.buttons.dynamic-button
+  (:require [quo.react-native :as rn]
+            [reagent.core :as reagent]
+            [status-im.i18n.i18n :as i18n]
+            [quo2.components.icon :as icon]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.markdown.text :as text]))
+
+(defn- in?
+  "true if coll contains elm"
+  [coll elm]
+  (some #(= elm %) coll))
+
+(defn- get-button-color [type pressed? customization-color]
+  (if (in? '(:jump-to :mention) type)
+    (if pressed?
+      (colors/custom-color-by-theme customization-color 60 50)
+      (colors/custom-color-by-theme customization-color 50 60))
+    (if pressed?
+      (colors/theme-colors colors/neutral-80-opa-80 colors/white-opa-80)
+      (colors/theme-colors colors/neutral-80-opa-70 colors/white-opa-70))))
+
+(defn- get-icon-and-text-color [type]
+  (if (in? '(:jump-to :mention) type)
+    colors/white
+    (colors/theme-colors colors/white colors/neutral-100)))
+
+(defn- icon-view [type]
+  [icon/icon
+   (case type
+     :jump-to           :main-icons2/jump-to
+     :mention           :main-icons2/mention
+     :notification-down :main-icons2/arrow-down
+     :notification-up   :main-icons2/arrow-up
+     :search-with-label :main-icons2/search
+     :search            :main-icons2/search
+     :bottom            :main-icons2/arrow-down)
+   {:size            12
+    :color           (get-icon-and-text-color type)
+    :container-style {:margin-top    6
+                      :margin-bottom 6
+                      :margin-left   (case type
+                                       :jump-to           0
+                                       :mention           8
+                                       :notification-down 2
+                                       :notification-up   2
+                                       :search-with-label 8
+                                       :search            6
+                                       :bottom            6)
+                      :margin-right  (case type
+                                       :jump-to           8
+                                       :mention           2
+                                       :notification-down 8
+                                       :notification-up   8
+                                       :search-with-label 4
+                                       :search            6
+                                       :bottom            6)}}])
+
+(defn dynamic-button
+  "[:dynamic-button opts]
+   opts
+   {:type                :jump-to/:mention/:notification-down/:notification-up/:search/:search-with-label/:bottom
+    :on-press            fn
+    :count               mentions or notifications count
+    :customization-color customize jump-to and mention button color}"
```

Comment:
not only related to this p.r but do we have a common name for this prop??
It's common across a lot of components so we should aim to standardize this name to some extent

Reply:
Hi @J-Son89, Thank you for reviewing the PR.
Yes, in some places this variable is just named `color`.
I don't have a strong opinion, but `customization-color` sounds better to me, as this is a customization color.

Reply:
yep I agree and let's keep an eye on future pr's to try to use this name where possible 👍